public/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>fluffy-octo-engine</title>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" type="image/x-icon" href="https://loopback.io/favicon.ico">
+
+  <style>
+    h3 {
+      margin-left: 25px;
+      text-align: center;
+    }
+
+    a, a:visited {
+      color: #3f5dff;
+    }
+
+    h3 a {
+      margin-left: 10px;
+    }
+
+    a:hover, a:focus, a:active {
+      color: #001956;
+    }
+
+    .power {
+      position: absolute;
+      bottom: 25px;
+      left: 50%;
+      transform: translateX(-50%);
+    }
+
+    .info {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%)
+    }
+
+    .info h1 {
+      text-align: center;
+      margin-bottom: 0;
+    }
+
+    .info p {
+      text-align: center;
+      margin-bottom: 3em;
+      margin-top: 1em;
+    }
+
+    @media (prefers-color-scheme: dark) {
+      body {
+        background-color: rgb(29, 30, 32);
+        color: white;
+      }
+
+      a, a:visited {
+        color: #4990e2;
+      }
+
+      a:hover, a:focus, a:active {
+        color: #2b78ff;
+      }
+    }
+  </style>
+</head>
+
+<body>
+  <div class="info">
+    <h1>fluffy-octo-engine</h1>
+    <p>Version 1.0.0</p>
+
+    <h3>OpenAPI spec: <a href="./openapi.json">/openapi.json</a></h3>
+    <h3>API Explorer: <a href="./explorer">/explorer</a></h3>
```

Comment:
:robot: OpenAI

The links to the OpenAPI spec and API explorer should be updated to use the correct base URL for the API. Consider changing them to something like `<a href="/api/openapi.json">/api/openapi.json</a>` and `<a href="/api/explorer">/api/explorer</a>` respectively.
```suggestion
<h3>OpenAPI spec: <a href="/api/openapi.json">/api/openapi.json</a></h3>
<h3>API Explorer: <a href="/api/explorer">/api/explorer</a></h3>
```

<!-- This is an auto-generated comment by OpenAI -->